core/src/autogluon/core/searcher/bayesopt/gpautograd/optimization_utils.py

```diff
@@ -0,0 +1,269 @@
+"""
+optimization_utils
+==================
+
+Wrapper of SciPy L-BFGS-B optimizer minimizing objectives given by autograd.numpy (np) 
+executors. The main issue is to map between parameter dictionaries of np.ndarray 
+(executor side) and a single flat np.ndarray (optimizer side).
```

Comment:
this comment could be clarified as it's unclear to me what executor/optimizer side refer to